### ISSUE TITLE
ECommerce-shop and taxamo (European laws VAT)

### ISSUE THREAD

**Issue by Josua2012** [burnbright/silverstripe-shop#335]
Jeremy, you're going to include support in your eshop for new VAT European laws?
PrestaShop, WooCommerce, Magento, .... have already built plugins for it.
http://www.woothemes.com/2014/12/handling-eu-vat-woocommerce/
Now, it is a must for a shopping cart.
The best way is through www.taxamo.com.
You will develop a plugin to use taxamo?
Thanks,
Best regards,
Jose A.

**Comment by jedateach** [comment]
Hi @Josua2012 - definitely having global tax support is going to give this module a wider audience. I'm all for having this kind of support added to the module, at least architecting it in such a way that taxamo could be added on as a module.

However, I personally don't have plans to undertake this work. @markguinn do you have/ know of any plans?

**Comment by jedateach** [comment]
Relates to #12

**Comment by Josua2012** [comment]
Jeremy, If I pay for the work of making taxamo support. How much could it cost?
Thanks,
Best regards,
Jose A.

**Comment by jedateach** [comment]
@Josua2012 that would require some figuring out time, which unfortunately I don't have. I'm also not in a position to do the work either. Happy to answer any questions from someone who will do the work however.

**Comment by markguinn** [comment]
No, we don't have any clients in Europe so we've not had this feature requested at all. Thanks for bringing this to our attention @Josua2012. 

On the one hand this seems like it would be pretty simple to implement as they give you a prewritten spec for what a plugin needs to do: https://www.taxamo.com/doc/dev_guide/plugin_spec/. I think it's possibly as simple as implementing a subclass of TaxModifier for a basic integration, especially when using stepped checkout, maybe also needing a few extra checkout steps created.

On the other hand, there's a lot of logic to digest there (dynamic v universal pricing, sms verification, payment integration, etc). So there's potential for this to take a lot of time to do it right.

I'm also happy to help in an advisory role but I don't see myself being able to do the work any time soon.

**Comment by Josua2012** [comment]
The problem is not only for European vendors, but for any foreign seller who wants to sell digital products in Europe. So any shopping cart must be prepared for this, otherwise you can not legally make sales to Europe. Thank you @markguinn .

**Comment by dpde** [comment]
Maybe https://github.com/mpociot/vat-calculator is an alternative to Taxamo.
